### PR TITLE
Upgrade to Excalidraw 0.7.0

### PR DESCRIPTION
**Pull request by grant-davidson**
Excalidraw no longer supports width/height properties.  It just fills its container.
I'm not sure what css variable --bottomSpacing is doing in app.css:

   bottom: var(--bottomSpacing);

Doesn't seem to make any difference to the app appearance.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -2,6 +2,7 @@
   --bottomSpacing: 140px;
 }
 
-.ClaymateApp .excalidraw .App-bottom-bar {
+.ClaymateApp {
+  height: 100vh;
   bottom: var(--bottomSpacing);
```
Comment (dai-shi) [inline src/App.css:7]:
```suggestion
```

 It's probably a leftover. If it doesn't do anything, let's remove it. I prefer to keep things simple.

**Comment by dai-shi** [comment]
Hmm, the error was inside excalidraw. Any idea for fix? Maybe mock nanoid?

**Comment by grant-davidson** [comment]
I've been trying that but haven't succeeded so far

**Review by dai-shi — APPROVED** [review]
Cool. You made it work. 🎉

**Pull request merged**